.github/actions/setup-job/action.yml

```diff
@@ -19,14 +22,43 @@ runs:
     - name: Install just
       uses: extractions/setup-just@aa5d15c144db4585980a44ebfdd2cf337c4f14cb #v1.4.0
       with:
-        just-version: '0.10.5'
+        just-version: ${{ env.JUST_VERSION }}
+
+    - name: Restore Rust toolchain cache
```

Comment:
This is not restoring the toolchain cache it cannot be called before the toolchain is there.